src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,132 @@
+package storage;
+
+import dukeexception.LoadException;
+import dukeexception.SaveException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+/**
+ * This class deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+
+    public String pathName;
+    public ArrayList<Task> taskList;
+    public static final String FILE_PATH = "/Users/jadetay/duke/data/tasks.txt";
```

Comment:
Correct me if I am wrong, but seems like this file path is redundant, since you have to specify the file path in the constructor